pkg/deploy/generator/const.go

```diff
@@ -28,3 +28,9 @@ const (
 	fileRPDevelopmentPredeploy = "rp-development-predeploy.json"
 	fileRPDevelopment          = "rp-development.json"
 )
+
+// Network security group constants
+const (
+	NSGControlPlaneSuffix = "-controlplane-nsg"
+	NSGNodeSuffix         = "-node-nsg"
+)
```

Comment:
I think it worth merging them with "Deployment const". Name is misleading "Deployments" as all of them are specific resource constants used in Deployments. some of those values are exported to use externally, some no.

Reply:
Please put these in pkg/util/subnet.